apps/storybook/src/HeatmapVis.stories.tsx

```diff
@@ -57,13 +57,13 @@ AxisValues.args = {
   domain,
   abscissaParams: {
     value: Array.from(
-      { length: dataArray.shape[1] }, // works even when right edge of last pixel is not provided
+      { length: dataArray.shape[1] }, // far edge of last pixel must not be included
       (_, i) => 100 + 10 * i
     ),
   },
   ordinateParams: {
     value: Array.from(
-      { length: dataArray.shape[0] + 1 },
+      { length: dataArray.shape[0] }, // far edge of last pixel must not be included
```

Comment:
The problem lies elsewhere: asserting length of `abscissaValue` and `ordinateValue` removed the possibility to give pixel edges as values which is something we supported earlier.

We should leave the stories as unchanged and instead remove the [`assertLength`](https://github.com/silx-kit/h5web/blob/main/packages/lib/src/vis/heatmap/HeatmapVis.tsx#L68_) as [`getAxisValues`](https://github.com/silx-kit/h5web/blob/main/packages/lib/src/vis/heatmap/utils.ts#L124) already asserts the length of arrays.